src/app/components/pages/send-skycoin/send-form/send-form.component.ts

```diff
@@ -147,7 +153,7 @@ export class SendFormComponent implements OnInit, OnDestroy {
 
     const parts = amountControl.value.toString().split('.');
 
-    if (parts.length === 2 && parts[1].length > 6) {
+    if (parts.length === 2 && parts[1].length > window['blockchainService'].currentMaxDecimals) {
```

Comment:
What's the reason to store blockchainService in a window object?

Reply:
When `validateAmount` is called, `this` is `undefined`, so checking `this.blockchainService.currentMaxDecimals` causes an error. Using `window` to temporarily store `blockchainService` is just a quick way to fix the problem.

Reply:
It's probably caused by missing `.bind(this)` here 
https://github.com/skycoin/skycoin-web/blob/7879a58ef06f1905829693a44453231f6aa8a8cd/src/app/components/pages/send-skycoin/send-form/send-form.component.ts#L136

Reply:
You are right. I changed it, thanks.